### PR TITLE
[Feature][Implement] Set initiating phase abstract feature

### PR DESCRIPTION
**Pull request by joowon-dm-snu**
### 🚀  Related Issue

Implementation PR for #26  

#### Before Implementation (Reviewers will check here)
 - [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
 - [ ] Did you write abstract codes for the new feature?
 - [ ] Did you add documentations for upcoming feature?
 - [ ] Did you write necessary tests?
 - [ ] Did you make sure your PR does only one thing?
 - [ ] Ready for implemtation?. Please ask for a review!

#### After Implementation (Reviewers will check here)
 - [ ] Did you verify new and existing tests pass locally with your changes?
 - [ ] Did you make sure to update the documentation with your changes? (if necessary)
 - [ ] Did you update the CHANGELOG? (not for typos, docs, test updates, or internal minor changes/refactorings)
 - [ ] Ready for merge?

### CODE REVIEW

File: manta_client/base/settings.py [hunk 1]
```diff
@@ -164,3 +178,10 @@ def freeze(self):
 
     def is_frozen(self):
         return self.__frozen
+
+    @property
+    def _offline(self) -> bool:
+        res = False
+        if self.mode == "offline":
+            res = True
+        return res
```
Comment (BlueWhaleKo) [inline manta_client/base/settings.py:187]:
```python
@property
def _offline(self) -> bool:
    return self.mode == "offline"
```

File: manta_client/sdk/manta_login.py [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+        pass
+
+
+def login(
+    api_key: str = None, base_url: str = None, settings: Settings = None, api: MantaAPI = None, silent: bool = None
+):
+    kwargs = dict(locals())
+
+    silent = kwargs.pop("silent", None)
+    _login = _MantaLogin(silent=silent)
```
Comment (BlueWhaleKo) [inline manta_client/sdk/manta_login.py:33]: silent is expected to be `bool` in `_MantaLogin`

**Review by BlueWhaleKo — COMMENTED** [review]
LGTM!

File: manta_client/base/settings.py (hunk 1 above)
Comment (joowon-dm-snu) [reply manta_client/base/settings.py:187]: originally there will be some more logics like <disable> mode. i'm still thinking about code there. thanks :)

**Pull request merged**